package.json

```diff
@@ -18,16 +21,24 @@
   },
   "homepage": "https://github.com/OlegMakarenko/symbol-statistic-servicet#readme",
   "dependencies": {
+    "@babel/core": "^7.9.0",
+    "@babel/node": "^7.8.7",
+    "@babel/preset-env": "^7.9.5",
     "axios": "^0.19.2",
+    "babel-eslint": "^10.1.0",
+    "babel-preset-env": "^1.7.0",
     "body-parser": "^1.18.3",
+    "cross-env": "^7.0.2",
     "express": "^4.14.0",
     "express-ws": "^4.0.0",
     "mongodb": "^2.2.16",
     "mongoose": "^5.5.8",
-    "nem2-sdk": "^0.16.3",
+    "symbol-data-lib": "file:symbol-data-lib",
```

Comment:
It can be installed directly from github
`"symbol-data-lib": "nemfoundation/symbol-data-lib"`

Reply:
`npm i nemfoundation/symbol-data-lib`
i tried, it's not working in my machine, that is error message.

i did a shell script `init_setup.sh` for this, you can have a look on it.

Reply:
Will take a look